scripts/rollup.config.js

```diff
@@ -27,6 +27,7 @@ const options = {
       }),
       module: defaults.module,
       test: Object.assign(defaults.test, {
+        '@sheerun/mutationobserver-shim': 'MutationObserver',
```

Comment:
This might need to be installed, I don't see it in package.json? Or is this part of another package?

Reply:
It's a deep dependency of `@testing-library/react`. Could probably use a comment...